pyClient/README.md

```diff
@@ -38,18 +35,7 @@ test_erc_token_mixing.py [ZKSNARK]
 where `[ZKSNARK]` is the zksnark to use (must be the same as the one used on
 the server).
 
-## Install solc manually
-
-This command might be necessary if the `py-solc-x` package cannot find `solc`
-and fails to fetch it (or fails to fetch the right version).
-
-```console
-# Download the solidity compiler to compile the contracts
-$ wget https://github.com/ethereum/solidity/releases/download/[solc-version]/[solc-for-your-distribution] -O $ZETH/pyClient/zeth-devenv/lib/[python-version]/site-packages/solcx/bin/solc-[solc-version]
-$ chmod +x $ZETH/pyClient/zeth-devenv/lib/[python-version]/site-packages/solcx/bin/solc-[solc-version]
-```
+## Note on solc compiler installation
 
-To run this command you need to replace the solidity version (denoted by
-`[solc-version]`), the python version (denoted `[python-version]`), and binary
-file (denoted by `[solc-for-your-distribution]`) by your system specific pieces
-of information.
+Note that `make test` will automatically install solidity compiler in `$HOME$/.solc`
```

Comment:
This makes it sounds as though one must run `make test` before using the library / commands.
Is the compiler not installed on-demand (as required)?  Maybe we can just say this?

Reply:
Yes, I'll modify it by removing `make test`.